src/Infrastructure/IO/src/makefile

```diff
@@ -6,11 +6,13 @@ SOURCEC   = ESMCI_IO_NetCDF.C ESMCI_IO.C ESMCI_IO_Handler.C ESMCI_IO_Gridspec.C
 SOURCEF   =
 STOREH    = ESMCI_IO_NetCDF.h ESMCI_IO.h ESMCI_IO_Handler.h ESMCI_IO_Gridspec.h ESMCI_IO_Scrip.h ESMCI_IO_YAML.h
 
-#ifeq ($(ESMF_PIO),internal)
-SOURCEC  += ESMCI_PIO_Handler.C
-# PIO_Handler not called directly (and would require PIO includes as well)
-#STOREH   += ESMCI_PIO_Handler.h
-#endif
+ifdef $(ESMF_PIO)
+  ifneq ($(ESMF_PIO),'none')
```

Comment:
@jedwards4b when would ESMF_PIO ever be set to 'none'?  I thought that if there was no PIO the ESMF_PIO would just be undefined.